app/views/pages/home.html.erb

```diff
@@ -1,7 +1,23 @@
 <div class="hero-section">
   <div class="hero-content">
-    <h1 class="hero-title">Title Title</h1>
-    <p class="hero-description">Description Description Description</p>
-    <button class="button">Call to Action</button>
+    <h1 class="hero-title">Support Change, Empower Communities</h1>
+    <p class="hero-description">Make a meaningful impact with your support</p>
+    <button class="button" onclick="window.location.href='#categories'">Get Started Today</button>
   </div>
 </div>
+
```

Comment:
I think we could already use the db to show the categories, this would make it easier and more reliable to link to the index pages for charity categories as well. So iterate over categories and  use category.name instead of title. Then we could link to category_charities_path(category)